src/newrelic_logging/auth.py

```diff
@@ -231,6 +235,12 @@ def validate_jwt_config(auth: dict) -> dict:
     if not auth['audience']:
         raise ConfigException('audience', 'missing JWT audience')
 
+    if auth['exp_offset'] < 0:
```

Comment:
Can it be 0? I would've thought `<= 0` here

Reply:
Oh but your error message just says "not be negative". So if that's what you mean, nevermind

Reply:
@jbeveland27 specifically want to prevent 0 minutes because I _think_ that will cause the same error that this PR is supposed to address #31 which is happening in that case due to a negative offset.  At 0, it would basically say the expiration is "now" rather than "5 minutes ago" (which is the source of the original problem).  So I think 0 is still not valid.

Reply:
@jbeveland27 realized what you were saying now.  I have changed the code to be `<=` and adjusted the error message.